harness/tests/common/test_api.py

```diff
@@ -19,10 +47,13 @@ def gen_make_url_cases() -> List[Case]:
         Case("http://localhost:8080/", "api/v1/experiments/", f"{host}/{path}/"),
         Case("http://localhost:8080/proxied/", "", f"{host}/proxied/"),
         Case("http://localhost:8080/proxied", "", f"{host}/proxied"),
-        Case("http://localhost:8080/proxied", "/api/v1/experiments", f"{host}/proxied/{path}"),
-        Case("http://localhost:8080/proxied", "api/v1/experiments/", f"{host}/proxied/{path}/"),
-        Case("http://localhost:8080/proxied/", "/api/v1/experiments", f"{host}/proxied/{path}"),
-        Case("http://localhost:8080/proxied/", "api/v1/experiments/", f"{host}/proxied/{path}/"),
+        # Case("http://localhost:8080/proxied", f"/{path}", f"{host}/proxied/{path}"),
+        # Case("http://localhost:8080/proxied", f"{path}/", f"{host}/proxied/{path}/"),
+        # Case("http://localhost:8080/proxied/", f"/{path}", f"{host}/proxied/{path}"),
```

Comment:
is there a reason to keep them commented?

Reply:
to show case this doesn't support these. I need to update the branch to make the diffs a bit more clear